tests/fixtures.ts

```diff
@@ -24,6 +26,20 @@ type PageFixtures = {
 };
 
 export const test = base.extend<PageFixtures>({
+  lendingBarAutoRenew: async ({ page }, use) => {
+    // Set up the fixture.
+    const lendingBarAutoRenew = new LendingBarAutoRenew(page);
+
+    await page.route(/(analytics|fonts)/, route => {
```

Comment:
QUESTION: Does this now need to include "athena"?

Reply:
actually we do not need to change anything here as we are using demo page